articles/active-directory/manage-apps/use-scim-to-provision-users-and-groups.md

```diff
@@ -820,7 +821,7 @@ To host the service within Internet Information Services, a developer would buil
    ```
 
 ### Handling endpoint authentication
-Requests from Azure Active Directory include an OAuth 2.0 bearer token.   Any service receiving the request should authenticate the issuer as being Azure Active Directory for the expected Azure Active Directory tenant, for access to the Azure Active Directory Graph web service.  In the token, the issuer is identified by an iss claim, like "iss":"https://sts.windows.net/cbb1a5ac-f33b-45fa-9bf5-f37db0fed422/".  In this example, the base address of the claim value, https://sts.windows.net, identifies Azure Active Directory as the issuer, while the relative address segment, cbb1a5ac-f33b-45fa-9bf5-f37db0fed422, is a unique identifier of the Azure Active Directory tenant for which the token was issued.  If the token was issued for accessing the Azure Active Directory Graph web service, then the identifier of that service, 00000002-0000-0000-c000-000000000000, should be in the value of the token’s aud claim.  Each of the applications that are registered in a single tenant may receive the same `iss` claim with SCIM requests.
+Requests from Azure Active Directory include an OAuth 2.0 bearer token.   Any service receiving the request should authenticate the issuer as being Azure Active Directory for the expected Azure Active Directory tenant, for access to the Azure Active Directory Graph web service.  In the token, the issuer is identified by an iss claim, like "iss":"https://sts.windows.net/cbb1a5ac-f33b-45fa-9bf5-f37db0fed422/".  In this example, the base address of the claim value, https://sts.windows.net, identifies Azure Active Directory as the issuer, while the relative address segment, cbb1a5ac-f33b-45fa-9bf5-f37db0fed422, is a unique identifier of the Azure Active Directory tenant for which the token was issued. The audience for the token will be the parent application id for the app in the gallery. The parent application id for all custom apps is 8adf8e6e-67b2-4cf2-a259-e3dc5476c621. The parent application id for each app in the gallery varies. Please contact ProvisioningFeedback@microsoft.com for questions on parent app id for a gallery application. Each of the applications that are registered in a single tenant may receive the same `iss` claim with SCIM requests.
```

Comment:
Please contact ProvisioningFeedback@microsoft.com for questions on **the** parent app id for a gallery application. 

Let's include parent app id label as it appears when you import the application from MSGraph, so that the reader knows what we are talking about.

Reply:
The parent application id for all **the** custom apps is 8adf8e6e-67b2-4cf2-a259-e3dc5476c621